src-built-in/components/toolbar/stores/toolbarStore.js

```diff
@@ -3,18 +3,25 @@
 * All rights reserved.
 */
 import async from "async";
-import * as menuConfig from '../config.json';
+import * as toolbarConfig from '../config.json';
 import * as storeExports from "../stores/searchStore";
 
 import { Actions as SearchActions } from "./searchStore"
 
+// this refers to the new menu section in the toolbar, to add backward compatibility
+// if this structure does not exist then just default to using the old array structure
+const menuConfig = 'menus' in toolbarConfig ? toolbarConfig.menus : toolbarConfig
```

Comment:
Should you use ' or "?

Reply:
" for consistency, thanks for spotting this @NinjaManatee 👍